### PR TITLE
Klimafragebogen

### PR DESCRIPTION
**Pull request by annatraussnig**

### CODE REVIEW

File: packages/styleguide/src/components/TeaserCarousel/Carousel.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ export const Carousel = ({
   color,
   grid,
   isSeriesNav,
+  borderRadius,
```
Comment (felixmichel) [inline packages/styleguide/src/components/TeaserCarousel/Carousel.js:32]: shoud be removed again

File: packages/styleguide/src/components/TeaserCarousel/Carousel.js [hunk 2]
```diff
@@ -51,6 +52,7 @@ export const Carousel = ({
         tileCount,
         tileMaxWidth,
         grid,
+        borderRadius,
```
Comment (felixmichel) [inline packages/styleguide/src/components/TeaserCarousel/Carousel.js:55]: this too

File: packages/styleguide/src/components/TeaserCarousel/Tile.js [hunk 3]
```diff
@@ -127,6 +127,7 @@ const Tile = ({
   color: tileColor,
   outline: tileOutlineColor,
   bigger: tileBigger,
+  borderRadius: tileBorderRadius,
```
Comment (felixmichel) [inline packages/styleguide/src/components/TeaserCarousel/Tile.js:130]: and also here, all the borderRadius stuff is not necessary anymore

**Review by felixmichel — CHANGES_REQUESTED** [review]
remove borderRadius props from carousel

**Comment by annatraussnig** [comment]
tested:

- shares (overview, question & person)
- sommerfragebogen (search is broken – also in prod: this is a separate issue – will be fixed tomorrow)
- postcard overview

**Pull request merged**